rllib/policy/eager_tf_policy.py

```diff
@@ -561,19 +557,21 @@ def _initialize_loss_with_dummy_batch(self):
                 dummy_batch["state_out_{}".format(i)] = h
 
             if self._state_in:
-                dummy_batch["seq_lens"] = tf.convert_to_tensor(
-                    np.array([1], dtype=np.int32))
+                dummy_batch["seq_lens"] = np.array([1], dtype=np.int32)
+
+            # Convert everything to tensors.
+            dummy_batch = tf.nest.map_structure(
```

Comment:
I'm not sure which versions of `tf` `rllib` supports nowadays, but I noticed some differences between tf1 and tf2 nest library. Might want to consider using the standalone [dm-tree](https://github.com/deepmind/tree), which also has a couple of additional functions that for some reason are not exposed in the `tf.nest` module.

Reply:
Ah, very cool. Was looking for something like this. Also good for torch, then.

Reply:
Done.